python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -55,6 +68,50 @@ def __init__(self, database, driver='sqlite', host=None, port=None):
         self.host = host
         self.port = port
         self.database = database
+        self._seeing_column = 'FWHMeff'
+
+        # a dict keyed on the user interface (i.e. args to getObservationMetaData)
+        # names of OpSim data columns.  Returns a tuple that is the
+        # (name of data in OpSim, transformation to go from user interface to OpSim units, dtyp in OpSim)
+        self._user_interface_to_opsim = {'obsHistID': ('obsHistID', None, np.int64),
```

Comment:
This is a pretty fundamental attribute containing hard coded information about the mapping of OpSim variables to the user interface. It might make sense to make this a more 'public' variable.